Jenkinsfile

```diff
@@ -175,6 +175,7 @@ pipeline {
         }
         stage('Run GFE Integeration Tests') {
             steps {
+                sh 'python SpiNNakerGraphFrontEnd/gfe_integration_tests/script_builder.py short'
```

Comment:
Does GFE have any "long" tests?

Reply:
Currently no so yes the short flag should go.